### ISSUE TITLE
warning when running tests

### ISSUE THREAD

**Issue by rngadam** [transist/echidna-spider#2]
```
bundle exec rspec spec/
```

warning:

```
/home/rngadam/.rvm/gems/ruby-1.9.3-p327-falcon@echidna-streaming/gems/goliath-1.0.1/lib/goliath/application.rb:95:in `run!': undefined method `camel_case' for Goliath::Application:Module (NoMethodError)
        from /home/rngadam/.rvm/gems/ruby-1.9.3-p327-falcon@echidna-streaming/gems/goliath-1.0.1/lib/goliath/application.rb:129:in `block in <module:Goliath>'
```

**Comment by flyerhzm** [comment]
fix in pull request #4

**Comment by 0x1b2c** [comment]
Merged, thanks Richard.
